pybandits/meta_model.py

```
@@ -0,0 +1,391 @@
+            if len(context) < len(actions):
+                raise ValueError(
+                    f"context has {len(context)} rows but {len(actions)} actions were provided; "
+                    "context must have at least as many rows as actions."
+                )
+            context = context[-len(actions) :]
+
+        rewards_dict: Dict[ActionId, List[Any]] = defaultdict(list)
+        context_dict: Dict[ActionId, List[Any]] = defaultdict(list)
+
+        if quantities is None:
+            for i, (a, r) in enumerate(zip(actions, rewards)):
```

Ruff (0.15.13)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)